### PR TITLE
sort netctl command names in help

### PR DESCRIPTION
**Pull request by rchirakk**

### CODE REVIEW

File: netctl/netctl/main.go [hunk 1]
```diff
@@ -2,16 +2,25 @@ package main
...
+
+func (a byName) Len() int           { return len(a) }
+func (a byName) Swap(i, j int)      { a[i], a[j] = a[j], a[i] }
+func (a byName) Less(i, j int) bool { return a[i].Name < a[j].Name }
+
 func main() {
 	app := cli.NewApp()
 	app.Flags = netctl.NetmasterFlags
 	app.Version = "\n" + version.String()
+	// use sort.Slice() in go1.8
```
Comment (gaurav-dalvi) [inline netctl/netctl/main.go]: wouldn't it break backward compatibility with earlier golang version ?

Comment (rchirakk) [inline netctl/netctl/main.go]: we can bump up the min. go version requirement to 1.8

Comment (unclejack) [inline netctl/netctl/main.go]: @rchirakk: We're not going to upgrade soon.

Comment (dseevr) [inline netctl/netctl/main.go]: Go 1.8.0 was a disastrous release with many bugs so I'd vote for skipping it 😄

Comment (rchirakk) [inline netctl/netctl/main.go]: @unclejack @dseevr how is 1.8.1.
The  comment in the code was to point that this PR can be 
changed to a single line in > go-1.8

Comment (dseevr) [inline netctl/netctl/main.go]: @rchirakk Yeah, that makes sense.  I might even make it a `TODO` so it's easier to find.

Based on the [bugs being fixed for 1.8.2](https://github.com/golang/go/milestone/54), it seems like 1.8.1 was an okay release

**Pull request merged**